File: Practica 10/Python/Hexagono.py
from Figura import Figura
import math

class Hexagono(Figura):
    def __init__(self, color: str, side: float):
        super().__init__(color)
        self.side = side
    
    def perimetro(self) -> float:
        return 6 * self.side
    
    def area(self) -> float:
        return (3 * math.sqrt(3) * self.side**2) / 2

File: Practica 10/Java/Hexagono.java
package Java;

public class Hexagono extends Figura {
    private double side;

    public Hexagono(String color, double side) {
        super(color);
        this.side = side;
    }

    @Override
    public double perimetro() {
        return 6 * side;
    }

    @Override
    public double area() {
        return (3 * Math.sqrt(3) * side * side) / 2;
    }
}
